pkg/columnstore/granule.go

```diff
@@ -12,22 +13,21 @@ import (
 )
 
 type Granule struct {
-	sync.RWMutex
 
 	// least is the row that exists within the Granule that is the least.
 	// This is used for quick insertion into the btree, without requiring an iterator
-	least Row
-	parts []*Part
+	least *Row
+	parts *PartList
 
 	// card is the raw commited, and uncommited cardinality of the granule. It is used as a suggestion for potential compaction
-	card int
+	card uint64
 
 	schema *Schema
 
 	granulesCreated prometheus.Counter
 
 	// pruned indicates if this Granule is longer found in the index
-	pruned bool
+	pruned uint64
```

Comment:
Before `pruned` was a simple bool for which the comment made sense. I don't really get what a number between 0 and maxUInt64 now says. Can we update the comment? Thanks!